### ISSUE TITLE
Running slow on android device

### ISSUE THREAD

**Issue by cjoshi85** [CityOfZion/neon-wallet-react-native#6]
The app is running very slow on the android devices. Can you please let me know how I can enhance the speed of scrypt library?

**Comment by ixje** [comment]
Yes this issue is known on the old repository ([here](https://github.com/ixje/neon-wallet-react-native/issues/1])). There is a native library that could replace the javascript `scrypt` implementation: https://github.com/Airbitz/react-native-fast-crypto

I helped another Android user out by testing its validity and it could be used. I just have not heard from that user if it solves the issue on Android devices. I only have a ios device so I can't test it myself.
